survey_form.css

```diff
@@ -0,0 +1,54 @@
+body {
+  font-family: Arial, sans-serif;
+  background-color: #bbb1f2;
+}
+
+header {
+  text-align: center;
+  margin-bottom: 20px;
+}
+
+#survey-form {
+  width: 80%;
+  margin: 0 auto;
+}
+
+label {
+  display: block;
+  margin-bottom: 5px;
+}
```

Comment:
Sir, this is my commit for the code 
once you accept the invitation then I can request for the Review